README.md

```diff
@@ -1,2 +1,80 @@
 # paperTemplate
+
 Template repository for scientific journal articles and associated materials
+
+## Input data
+
+Use the `data-raw` folder to save code that extracts or downloads the input
+data used for your paper. This code can output data to the `data` directory.
+
+Use the `data` directory to store input data. In this template, `.rds` files
+in the `data` directory will not be tracked with git (see `.gitignore` file).
+
+## Analysis
+
+Use the `analysis` directory to save analysis code. Analysis should be split
+into separate scripts appropriately. Operations that are common across analysis
+scripts can be functionalized and saved to the `utils.R` file to minimize
+repeated code.
+
+Outputs of analysis can also be saved to the `data` directory as `.rds` files, 
+or a new `results` directory can be created, as needed.
+
+## Writing
+
+Rmarkdown files are useful for paper writing, because code can be embedded
+within, making number-plugging easier and more reproducible.
+
+This template includes three Rmarkdown files:
+1. manuscript
+2. tables_figures
+3. appendix
+
+It may be appropriate to combine the tables and figures with the manuscript,
```

Comment:
To reduce runtime another strategy could be to have a script or Rmarkdown file that produces jpg figures or csv tables, which we then include in the manuscript Rmarkdown file.

Reply:
I see that you are kind of doing this with the figures section too. I guess difference is functions vs saving output separately. I think either is good depending on the use case